### ISSUE TITLE
Would it be possible to create submenus for all NAM entries?

### ISSUE THREAD

**Issue by ParanormalBanana** [memo33/submenus-dll#1]
I'm just a guy getting back into SC4 and I found NAM interesting but near impossible to navigate with the endless scroll lists and obtuse naming and icons. Then I found the submenus mod which would be the perfect solution for all of these problems. Unfortunately, it seems there aren't any NAM submenus built in. 
Is there a way that I could do it? Don't have much experience in modding but I've got time tho.

**Comment by memo33** [comment]
Yes, it's possible. Eventually I think NAM submenus will be integrated into the NAM itself, but I'm not sure when. The process involves updating all the puzzle piece button Exemplar files with properties `Item Submenu Parent ID` and `Item Button Class`, as described for network items in the Readme file. A couple new submenu icons may be needed as well.

If you're thinking of replacing each tab loop item by an individual submenu item, that's theoretically possible as well, but requires a complete restructuring of the NAM-internal RUL0 file or maybe further changes of the DLL. It would also require many many new icons, so this is probably not going to be implemented for a while.

**Comment by memo33** [comment]
Closing this, as there's now a separate project for this: https://community.simtropolis.com/forums/topic/763671-network-addon-mod-submenu-open-alpha/
